### PR TITLE
Fix backup deletion on windows

### LINKED ISSUE #3939: 9.2.1 on windows: backups can not be deleted
Dear GLPI user.

BEFORE SUBMITTING YOUR ISSUE, please make sure to read and follow these steps :

* We don't support community plugins. Contact directly their authors, or use the community forum : http://forum.glpi-project.org.
* For feature requests or enhancements, use the suggest dedicated site (http://suggest.glpi-project.org ). We check it very often.
* We prefer to keep this tracker in ENGLISH. If you want support in your language, the community forum (http://forum.glpi-project.org) is the best place.
* Please use the below template.
* Delete this text before submiting your issue.

The GLPI team.

------------
* Steps to reproduce (which actions have you made) :
 Clic Administracion/Clic Mantenimiento/Clic Borrar Permanentemente

* Expected result :
Borrar un backup antiguo

* Actual result :
Se refresca la pagina y no se borra el backup.

* URL of the page :
192.168.10.56/glpi

* Screenshot of the problem (if pertinent) :
image
image

* Your GLPI setup (you can find it in Setup > General menu, System tab) :

9.2.1 Colombia SurAmerica

image

### PR DESCRIPTION
**Pull request by AnaelMobilia**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #3939 

Fix path calculation - on Windows the path can be written with \ and / ... => problems on path validation (on line 581)
Fix #3939

### CODE REVIEW

File: front/backup.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
 Session::checkRight("backup", READ);
 
 // full path
-$path = GLPI_DUMP_DIR;
+$path = realpath(GLPI_DUMP_DIR);
```
[the remark this answers is not in the document]
Comment (cedric-anne) [reply front/backup.php]: Yes, `On windows realpath() will change unix style paths to windows style.` as the official PHP documentation says (http://php.net/manual/en/function.realpath.php).

[the remark this answers is not in the document]
Comment (trasher) [reply front/backup.php]: OK, you're right

**Comment by AnaelMobilia** [comment]
@trasher I can't change the target here too, have you rights to made this ?

Regards

**Comment by trasher** [comment]
Changed to target 9.2/bf

**Comment by AnaelMobilia** [comment]
Too much problem on windows to rebase due to a filename case change... Made a new PR #3987 

Regards

**Pull request closed without merging**